.github/workflows/deploy-indexer.yaml

```diff
@@ -77,6 +77,12 @@ jobs:
       RAILWAY_TOKEN: ${{ secrets.RAILWAY_TOKEN }}
     steps:
       - uses: actions/checkout@v3
+      - name: Install Node.js and pnpm
```

Comment:
The workflow is installing Node.js and pnpm but then attempting to run 'pnpm install' without ensuring the PATH is updated or that pnpm is properly available. Additionally, installing dependencies without using them in subsequent steps may indicate incomplete implementation. Verify that the installed dependencies are actually needed for the 'Prune DB inactive schemas' step, or this could cause the workflow to fail.